example/passthrough_ll.c

```diff
@@ -336,6 +377,13 @@ static int lo_do_lookup(fuse_req_t req, fuse_ino_t parent, const char *name,
 		inode->fd = newfd;
 		inode->ino = e->attr.st_ino;
 		inode->dev = e->attr.st_dev;
+#ifndef HAVE_AT_EMPTY_PATH
+		struct lo_inode *parent_inode;
+
+		parent_inode = lo_inode(req, parent);
+		snprintf(inode->pathname, sizeof(inode->pathname), "%s/%s",
```

Comment:
This will get out of sync if the parent directory is renamed, even if the rename comes through the mountpoint.

Could we store only the directory entry name here, and reconstruct the full path on demand?